vividus-plugin-rest-api/src/main/resources/org/vividus/http/attachment/api-message.ftl

```diff
@@ -94,6 +101,24 @@
                 </div>
             </div>
         </#if>
+
+        <#if curlCommand??>
+            <div class="panel panel-info">
+                <div class="panel-heading">
+                    <h4 class="panel-title toggleable">
+                        <a data-toggle="collapse" data-target="#collapse-curl" href="#collapse-curl" class="collapsed">cURL command</a>
+                    </h4>
+                </div>
+                <div id="collapse-curl" class="panel-collapse collapse">
+                    <div class="container">
+                        <pre><code class="language-shell">${curlCommand}</code></pre>
+                        <button class="copy-button" title="Copy to clipboard" onclick="copyCurlCommand()">
+                        	<img src="../../webjars/bootstrap/3.4.1/fonts/clipboard.svg">
```

Comment:
is there any indicator that content is copied?

Reply:
<img width="1376" alt="Screenshot 2024-01-18 at 23 15 12" src="https://github.com/vividus-framework/vividus/assets/35694955/016eb017-a4cf-4ae8-b178-0599136b09ff">